src/status_im/transport/inbox.cljs

```diff
@@ -115,9 +122,23 @@
                            #(log/info "offline inbox: request-messages response" %)
                            #(log/error "offline inbox: request-messages error" %1 %2 to from))))
 
+(re-frame/reg-fx
+ ::request-history
+ (fn [{:keys [wnode topics to from sym-key-id web3]}]
+   (request-inbox-messages web3
+                           wnode
+                           topics
+                           to
+                           from
+                           sym-key-id
+                           #(log/info "offline inbox: request-messages response" %)
```

Comment:
Probably not part of this PR, but current log information print topics being requested, but not `from` / `to` parameters. Would be helpful to log all the parameters being used for request.

Reply:
I will take care of this when doing https://github.com/status-im/status-react/issues/4387